dashboard/README.md

```diff
@@ -1,70 +1,161 @@
+  
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+```bash
+ .
+ ├── public									# HTML entry
+ ├── server 								# endpoint configuration
+ └── src
+     ├── actions							# redux actions
+     ├── assets
+     │   ├── constants						# constants used in code
+     │   ├── images							# images
+     │   └── logo							# Pbench logos
+     ├── modules
+     │   ├── components						# functional components
+     │   └── containers						# parent components
+     ├── reducers							# redux reducer
+     ├── store								# redux store
+     └── utils								# utility functions
+```
 
-### `npm test`
+  
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Assets
 
-### `npm run build`
+  
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+  
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Prerequisites
 
-### `npm run eject`
+  
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Install Node JS
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Refer [https://nodejs.org/en/] (Node.js) official website
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+  
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Install create-react-app
 
-## Learn More
+Install create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+  
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+```bash
 
-### Code Splitting
+$ npm install -g create-react-app
```

Comment:
Do you actually need `create-react-app` installed to build, debug, and run the dashboard? I thought that was just the command to create a new React template? (That is, we did this once, but don't need or even want to do it again.)

Reply:
Yes, not required. Removed it